pytorch_lightning/strategies/hpu_parallel.py

```diff
@@ -47,9 +48,15 @@ def __init__(
         self,
         accelerator: Optional["pl.accelerators.accelerator.Accelerator"] = None,
         parallel_devices: Optional[List[torch.device]] = None,
+        cluster_environment: Optional[ClusterEnvironment] = None,
         checkpoint_io: Optional[CheckpointIO] = None,
         precision_plugin: Optional[PrecisionPlugin] = None,
+        ddp_comm_state: Optional[object] = None,
+        ddp_comm_hook: Optional[callable] = None,  # type: ignore
+        ddp_comm_wrapper: Optional[callable] = None,  # type: ignore
+        model_averaging_period: Optional[int] = None,
         process_group_backend: Optional[str] = "hccl",
+        **kwargs: Union[Any, Dict[str, Any]],
```

Comment:
The convention is this:

```suggestion
        **kwargs: Any,
```